### ISSUE TITLE
Word 'null' inserted at the start of file names.

### ISSUE THREAD

**Issue by autumn-lovelace** [leoding86/webextension-pixiv-toolkit#30]
This issue occurs for me when using custom filenaming for ugoira (via the extensions settings.) 
My specific pattern is: {author}-{authorId} {title}-{id}

An example of output is: 

> nullTheJuneBug_1802419__Cardigan_79553655.webm

from this page: https://www.pixiv.net/en/artworks/79553655

System info is:
**Pixiv Toolkit Version:** 3.7.3-20012112
**Pixiv Omina Version:** 0.4.1
**Operating System:** Windows 10 Home x64
**Browser:** Chrome Version 80.0.3987.106 (Official Build) (64-bit)

**Comment by leoding86** [comment]
Some feedbacks mentioned this issue, but I cannot reproduce it somehow. The only suggestion I can give to you is reinstall the extension.

**Comment by autumn-lovelace** [comment]
This issue didn't start happening originally - when I first set a custom naming scheme for files it was saving just fine.
When I went back and edited the scheming later, it started giving me the words 'null' at the start of the file name. Now even after reinstalling - this issue isn't resolving itself anymore.
The symptom seems fairly easy to guess - if you concatenate a null value, it is turned into the string literal "null". Somewhere in your file naming procedure, a null value is able to sneak in - how it does probably doesn't matter too much assuming this issue is mostly cosmetic.

I'd venture that checking for and discarding null values before concatenating them would clear this up.
